src/components/notifications/YZJ.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="mb-3">
+        <label for="yzjWebHookUrl" class="form-label">{{ $t("YZJ Webhook URL") }}<span style="color: red;"><sup>*</sup></span></label>
+        <input id="yzjWebHookUrl" v-model="$parent.notification.yzjWebHookUrl" type="url" class="form-control" required />
+        <i18n-t class="form-text" keypath="wayToGetTeamsURL">
+            <a href="https://www.yunzhijia.com/opendocs/docs.html#/tutorial/index/robot" target="_blank">{{ $t("here") }}</a>
+        </i18n-t>
+    </div>
+    <div class="mb-3">
+        <label for="yzjType" class="form-label">{{ $t("YZJ Robot Type") }}<span style="color: red;"><sup>*</sup></span></label>
+        <select id="yzjType" v-model="$parent.notification.yzjType" class="form-control" required>
+            <option :value="0">{{ $t("YZJ General Robot") }}</option>
+        </select>
+    </div>
+    <div class="mb-3">
+        <label for="yzjToken" class="form-label">{{ $t("YZJ Robot Token") }}<span style="color: red;"><sup>*</sup></span></label>
+        <hidden-input id="yzjToken" v-model="$parent.notification.yzjToken" type="text" class="form-control" required />
```

Comment:
Lets not be special here..
Currently, the `form-control` would be adding the weird halo..

```suggestion
        <HiddenInput id="yzjToken" v-model="$parent.notification.yzjToken" :required="true" autocomplete="new-password"></HiddenInput>
```